modules/reference/pages/diff/mp-50-60-diff.adoc

```diff
@@ -250,6 +251,18 @@ For more information about the OpenTelemetry API, see the link:https://www.javad
 For more information about migrating to the OpenTelemetry API, see link:https://opentelemetry.io/docs/migration/opentracing/#step-2-progressively-replace-instrumentation[the OpenTelemetry documentation].
 
 
+=== Use MicroProfile Telemetry and MicroProfile OpenTracing in the same trace system
+
+If a service in your microservice application is not ready to migrate the trace system from MicroProfile OpenTracing to MicroProfile Telemetry, you can still use MicroProfile Telemetry with your other services. To use both MicroProfile OpenTracing and MicroProfile Telemetry in the same trace system, set a compatible propagator to enable each service to send and receive spans from other services.
+
+Propagators are mechanisms that enable the movement of telemetry data among different services by using HTTP headers.
+Several propagator standards are available to move span information among microservices. The MicroProfile Telemetry `otel.propagators` configuration property configures which propagators are used. For a list of the propagators that OpenTelemetry supports, see the <<#select,Selecting a compatible propagator>> section.
+
+The following diagram shows two services that produce traces that are collected and exported to Zipkin. Service A is traced with MicroProfile OpenTracing, while service B is traced with MicroProfile Telemetry. When a compatible propagator is enabled, spans from each service can be incorporated in the distributed traces that are sent to Zipkin without losing the original parent-child relationship in the spans.
```

Comment:
Acrolinx suggests replacing spans with span.

Reply:
acrolinx wrongly thinks "span" is a verb here